src/redux/store.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
+});
+
+export default store;
```

Comment:
- [ ] Could you please use the book and category reducers in this file by importing them and using them in the reducer object as required?  You've already created them and using them here makes them available outside where you're using any of the actions 👍🏽

```jsx
import booksReducer from "./books/booksSlice"
import categoriesReducer from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})

export default store
```